src/main/java/com/unascribed/fabrication/mixin/f_balance/use_items_while_riding/MixinMinecraftClient.java

```diff
@@ -0,0 +1,23 @@
+package com.unascribed.fabrication.mixin.f_balance.use_items_while_riding;
+
+import com.unascribed.fabrication.support.EligibleIf;
+import com.unascribed.fabrication.support.Env;
+import com.unascribed.fabrication.support.MixinConfigPlugin;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.ClientPlayerEntity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(MinecraftClient.class)
+@EligibleIf(configEnabled="*.use_items_while_riding", envMatches=Env.CLIENT)
+public class MixinMinecraftClient {
+	@Redirect(method="doAttack()V", at=@At(value="INVOKE", target="Lnet/minecraft/client/network/ClientPlayerEntity;isRiding()Z"))
```

Comment:
Ditto.

Reply:
While i'm assuming this is done with At("JUMP"). i can't figure out how.